src/protocol_nmea.cpp

```diff
@@ -1663,7 +1663,7 @@ int decodeGSV(char* a, int aSize)
 
     int msgNum = NMEA_MSG_ID_GNGSV_START;
     
-    if(a[1] == 'x' || a[1] == 'X')        return NMEA_MSG_ID_GxGSV;
+    if(a[1] == 'x' || a[1] == 'X')        return NMEA_MSG_ID_GNGSV;
```

Comment:
Is this still appropriate?  Below, on 1787, we also have a GxVTG, that won't be parsed if this line remains.  Should that GxVTG also be updated to be GNVTG?